modules/ROOT/pages/introduction/intro-connector-configuration-overview.adoc

```diff
@@ -0,0 +1,148 @@
+= Anypont Connector Configuration Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Because a connector is a component in a Mule app, you should understand how to configure a Mule app before you try to configure a connector.
+
+== Connector Configuration Requirements
+
+Every connector has specific configuration requirements, which are described in the reference guide for that connector. Although connectors have different configuration requirements, most connector configurations require that you configure:
+
+* An input source to initiate the app flow
+* The authorization credentials needed to connect to the connector's data source, for example, to the Salesforce Marketing Cloud, Amazon S3, or SAP Concur APIs. Most connectors use the Basic (username and password), OAuth, or OAuth2 authentication schemes.
+* A connector operation to perform the desired action
+
+=== Configure an Input Source
+
+To configure an input source for a connector, select an input source associated with the connector you are configuring, or select the HTTP Listener operation or Scheduler endpoint. Not all connectors have associated input sources, but you can use the HTTP Listener operation or Scheduler endpoint as input sources for these connectors.
+
+For more information, see:
+
+* <<introduction/introduction-to-anypoint-connectors.adoc#input-sources, Input Sources>>
+* <<http/http-connector.adoc#input-sources, HTTP Connector>>
+* xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint (Trigger)]
+
+To determine whether the connector you are working with has its own input sources, see the reference guide for that connector.
+
+=== Authorization credentials
+
+Connectors that connect to external APIs use the Basic (Username and Password), OAuth, or OAuth2 authentication scheme.
+
+To configure authorization credentials for a connector, it is recommended that you create a <<global-element,global element>>, which enables you to reuse the configuration for other connector instances. For some connectors, configuring a global element is mandatory.
```

Comment:
```suggestion
To configure authorization credentials for a connector, create a <<global-element,global element>>, which enables you to reuse the configuration for other connector instances.
```
We should write under the assumption that our recommended practice is the one that we document.
You can modify this, if you like, with something like "Because some connectors require it, create...."